torch/csrc/inductor/aot_inductor_interface.h

```diff
@@ -99,4 +103,8 @@ AOTInductorError AOTInductorModelContainerGetMaxOutputShape(
     size_t output_idx,
     AOTInductorParamShape* output_shape);
 
+// Reads the constants used by the model from a file and stores it in memory.
+AOTInductorError AOTInductorModelContainerSetConstants(
+    std::map<std::string, at::Tensor>& constantTensors);
+
```

Comment:
A couple of concerns:

* this file is supposed to be a C interface, so we cannot have C++ constructs here
* to let the model container load the constants from the constant file, we could either:

    - change the ```AOTInductorModelContainerCreate``` to make it take one more argument (e.g. ```const char *constant_file```) and pass it down to InductorModelContainer's constructor to load the constants, or;
    -  add a new API like ```AOTInductorError AOTInductorModelContainerLoadConstants(const char *constant_file```) to explicitly load the constants.